src/Cake.Common/Tools/VSTest/VSTestSettings.cs

```diff
@@ -73,6 +73,13 @@ public sealed class VSTestSettings : ToolSettings
         /// </summary>
         public FilePath Diag { get; set; }
 
+        /// <summary>
+        /// Gets or sets the result directory.
+        /// Test results directory will be created in specified path if not exists.
+        /// VSTest.Console.exe flag <see href="https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options#ResultDirectory">/ResultsDirectory</see>.
+        /// </summary>
+        public DirectoryPath ResultsDirectory { get; set; }
```

Comment:
:+1: Matches command-line argument in VSTest-Console [docs](https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options#ResultDirectory)

[![image](https://user-images.githubusercontent.com/177608/108090522-114f3e80-7051-11eb-8000-0e8ad6d67adb.png)](https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options#ResultDirectory)